src/Microsoft.DotNet.CodeFormatting.Tests/Rules/MarkReadonlyFieldTests.cs

```diff
@@ -384,6 +384,21 @@ public void M(C1 c)
             Verify(Original(text), Readonly(text), true, LanguageNames.CSharp);
         }
 
+        [Fact]
+        public void MarkReadOnlyDoNotAnalyzeVisualBasicCode()
```

Comment:
Is this rule not applicable for VB? Isn't this test going to break when we implement this rule for VB? A more generic test might be to have a test-only analyzer that is csharp only and test that on VB code.